### PR TITLE
Fix #199, dosecubes not being recognized.

### LINKED ISSUE #199: cubeslice thinks dose cubes are LET cubes
and plots a LET bar, even if the data are dose.

### PR DESCRIPTION
**Pull request by nbassler**
Tested with TST001 from neptun.phys.au.dk/TRiP

Closes #199

### CODE REVIEW

File: pytrip/utils/cubeslice.py [hunk 1]
```diff
@@ -47,17 +47,18 @@ def load_data_cube(filename):
...
-    if data_header is not None:
-        basename_cube = os.path.splitext(data_header)[0]
-        d = pt.LETCube()
-
-    # try to load DOS cube
-    data_header, _ = pt.DosCube.parse_path(filename)
-    if d is None and data_header is not None:
-        basename_cube = os.path.splitext(data_header)[0]
-        d = pt.DosCube()
+    if "dosemlet" in filename.lower():
```
Comment (grzanka) [inline pytrip/utils/cubeslice.py]: What about fixing also (first?) https://github.com/pytrip/pytrip/issues/66 ?

extension is stored in pytrip, see https://github.com/pytrip/pytrip/blob/master/pytrip/let.py#L38

File: pytrip/utils/cubeslice.py [hunk 2]
```diff
@@ -47,17 +47,18 @@ def load_data_cube(filename):
...
+        # try to load LET cube    
+        data_header, _ = pt.LETCube.parse_path(filename)
```
Comment (grzanka) [inline pytrip/utils/cubeslice.py]: parse_path takes care about extension, see docstring here:
https://github.com/pytrip/pytrip/blob/master/pytrip/cube.py#L499

but the extension has to be set correctly.

File: pytrip/utils/cubeslice.py (hunk 1 above)
Comment (nbassler) [inline pytrip/utils/cubeslice.py]: I have no solution to #66 other that what I just did here.

File: pytrip/utils/cubeslice.py (hunk 2 above)
Comment (nbassler) [inline pytrip/utils/cubeslice.py]: hey leszek, I dont think we are time effective now, if you can you may directly edit the branch as you please.

Comment (nbassler) [inline pytrip/utils/cubeslice.py]: done

**Pull request merged**